models/user.go

```diff
@@ -40,3 +40,16 @@ func UserEmailAlreadyExist(email string) (userExist bool) {
 
 	return ormer.QueryTable(User{}).Filter("Email", email).RelatedSel().Exist()
 }
+
+// FindUserByEmail retrieves User by Email and returns error if User with given Email doesn't exist.
+func FindUserByEmail(email string) (user *User, err error) {
```

Comment:
In terms of naming, this is inconsistent with `GetUserById`. So pick between "Get" and "Find".

Reply:
@olivierobert ✅  renamed on [97b87ed](https://github.com/carryall/go-google-scraper-challenge/pull/26/commits/97b87edebed49f006f7fa01c62ca1cff7e592986)